test/PanelGroupSpec.js

```diff
@@ -111,9 +128,9 @@ describe('<PanelGroup>', () => {
       assert.equal(panelGroup.getAttribute('role'), 'tablist');
     });
 
-    it('Should provide each header tab with role of tab', () => {
-      assert.equal(links[0].getAttribute('role'), 'tab');
-      assert.equal(links[1].getAttribute('role'), 'tab');
+    it('Should provide each header tab with role of button', () => {
+      assert.equal(links[0].getAttribute('role'), 'button');
+      assert.equal(links[1].getAttribute('role'), 'button');
```

Comment:
this should still be 'tab', its intentionally making sure that the `tab` role is set on the headings